user_modules/dummy_connecttest.lua

```diff
@@ -64,8 +64,6 @@ function module.hmiConnection:EXPECT_HMIRESPONSE(id, args)
       end
       if results_args2 and results_args2.method then
         results_args2 = table.removeKey(results_args2, 'method')
-      elseif results_args2 and results_args2.data.method then
-        results_args2 = table.removeKey(results_args2.data, 'method')
       end
 
```

Comment:
@dboltovskyi , dummy_connecttest has based on connecttest, that is still used in many scripts and potentially can be used by any one in open source world.
Please, provide the same correction for connecttest.lua in sdl_atf - there is no such pull request.

Reply:
@Itileda In connecttest module the code is slightly different and this issue is not in place